admin/settings/class-admin-apple-settings-section.php

```diff
@@ -261,6 +261,9 @@ public function render_field( $args ) {
 			if ( $this->is_multiple( $name ) ) {
 				$multiple_name = '[]';
 				$multiple_attr = 'multiple="multiple"';
+				$size = min( $size, count( $type ) );
+			} else {
+				$size = 1;
 			}
```

Comment:
```suggestion
			$size = 1;
			if ( $this->is_multiple( $name ) ) {
				$multiple_name = '[]';
				$multiple_attr = 'multiple="multiple"';
				$size = min( $size, count( $type ) );
			}
```

Reply:
`$size` is actually already defined above this block, so we're overriding the value here under certain circumstances. If this was the first time we were defining `$size`, I would agree with you, that you define a default and then override it. But since this code is doing the overriding, I think it reads cleaner to have it be in an if/else, and if the is_multiple check is true, then it avoids updating the value and then updating it again.

Thanks for the review!